deployments/kovan/usdc/roots.json

```diff
@@ -3,5 +3,5 @@
     "configurator": "0xDb3c6Ae44FE4689f142Ed8dE1a87304249d3d5a6",
     "rewards": "0xC694877D91A8aEfb9D95cf34393cdC0DDdAded18",
     "fauceteer": "0xA42d20100750b4E4d777bD067134423C0775B869",
-    "bulker": "0x258fb42cCe70fC0EbAA00bFe9857eAdD41E56FA9"
+    "bulker": "0x37b29E24de3AE3a71665d3217f8DEc846f82C505"
```

Comment:
This the most up-to-date `Bulker` deployed by 🥈 last week in #549.

Reply:
This is no longer applicable after rebasing on main, which recently deployed a new Kovan Bulker in #548